client/lib/analytics/ad-tracking/load-tracking-scripts.js

```diff
@@ -104,6 +105,10 @@ function getTrackingScriptsToLoad() {
 		scripts.push( GOOGLE_GTM_SCRIPT_URL + TRACKING_IDS.akismetGoogleTagManagerId );
 	}
 
+	if ( mayWeInitTracker( 'parsely' ) ) {
```

Comment:
Do we need to init this here? Since we're only using it for tracking purchases, and we already chain the conversion tracking after the promise is resolved. 

The above would be a good idea if we were to load Parsely to get page views, but since Parsely seems pretty WordPress dependant, and it's mostly for content analytics, I think it makes more sense to just report back purchases. Potentially also signups and other deeper funnel events in a follow-up. And ignore the Calypso page views/route changes.

Reply:
Thanks, @gmovr, that makes perfect sense, we should load Parsely on demand for tracking purchases (for now) instead of loading the script by default. I've updated the PR